f8a_report/helpers/ingestion_helper.py

```diff
@@ -50,3 +54,26 @@ def ingest_epv(missing_latest_nodes):
         logger.error("Error while ingesting missing versions {}".format(e))
 
     return result
+
+
+def generate_ingestion_report(start_date: str, end_date: str, frequency='daily') -> Dict:
```

Comment:
@prashbnair This function acts as an Entry Point to the Ingestion reporting. 
I rewrote this just to match v2 standards of moving RDS query functions to `db_gateway`